src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -162,7 +162,7 @@ void addPassesNNPA(mlir::OwningOpRef<mlir::ModuleOp> &module,
 
   if (emissionTarget >= EmitLLVMIR)
     // Lower the remaining Krnl and all ZLow ops to LLVM dialect.
-    addKrnlToLLVMPasses(pm, /*enableCSE=*/true, verifyInputTensors);
+    addKrnlToLLVMPasses(pm, /*enableCSE=*/true);
```

Comment:
Not the main change in this patch. `verifyInputTensors` is redundant to pass in because we can get it directly from the options.